react/Provider/provider.tsx

```diff
@@ -11,8 +13,16 @@ import getCategoryName from '../queries/getCategoryName.gql'
 import getSpecificationName from '../queries/getSpecificationName.gql'
 import Context from '../Context/context'
 
+const BUTTON_CHOICE_IS_IMAGEM = 1
+const BUTTON_CHOICE_IS_TEXT = 2
+const BUTTON_CHOICE_IS_HTML = 3
+
+const NOT_SHOW_ALERT = 0
+const SHOW_ALERT_SAVE = 1
+const SHOW_ALERT_ERROR = 2
```

Comment:
Just a suggestion to improve even more:

```suggestion
const enum ButtonOptions = {
  image = 1,
  text = 2,
  html = 3,
}

const enum ShowAlertOptions = {
  notShow = 0,
  alertSave = 1,
  alertError = 2,
}
```

But as it is used in different files ([here](https://github.com/vtex-apps/badges/pull/4/files#diff-a63028146fdca5102f21f66d2f75af7b609fe825c7e21fa1c035c3f4b21a9424R36) and [here](https://github.com/vtex-apps/badges/pull/4/files#diff-a63028146fdca5102f21f66d2f75af7b609fe825c7e21fa1c035c3f4b21a9424R51), for example), these `enums`can be declared as global types. See this declaration example [here](https://github.com/vtex/support-center-admin/blob/fix%2Fsatisfaction-bug/react/typings/enums/satisfactionQuestionAnswers.ts) and using it [here](https://github.com/vtex/support-center-admin/blob/fix%2Fsatisfaction-bug/react/components/questions/ThumbIcons.tsx), please. What do you think?